torch/_inductor/config.py

```diff
@@ -76,6 +76,9 @@
 # enable pattern match with batch fusion (using torch op)
 batch_fusion = True
 
+# Find more opportunities for optimizing the graph (Experimental)
+optimus_opportunity_finder = False
```

Comment:
How would an end user be expected to use this?  It would seem only to be useful for PyTorch developers.

Reply:
Next, we will add "optimus_opportunity_finder" flag to the frontend config, where users could easily use this feature by setting it to True.